marketing_swag/demo/res_partner.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="res_partner_10" model="res.partner" >
+        <field name="name">Tina Williamson</field>
+        <field name="display_name">Tina Williamson</field>
+        <field name="category_id" eval="[(6, 0, [])]" />
+        <field name="property_stock_customer" ref="stock.stock_location_customers" />
+        <field name="partner_share" eval="True" />
+        <field name="property_stock_supplier" ref="stock.stock_location_suppliers" />
+        <field name="calendar_last_notif_ack">2023-07-28 07:19:15</field>
+        <field name="image_1920" type="base64" file="marketing_swag/static/src/binary/res_partner/10-image_1920" />
+    </record>
+    <record id="res_partner_11" model="res.partner" >
+        <field name="name">B&amp;C</field>
+        <field name="display_name">B&amp;C</field>
+        <field name="category_id" eval="[(6, 0, [])]" />
+        <field name="property_stock_customer" ref="stock.stock_location_customers" />
+        <field name="partner_share" eval="True" />
+        <field name="property_stock_supplier" ref="stock.stock_location_suppliers" />
+        <field name="calendar_last_notif_ack">2023-07-28 07:26:45</field>
+        <field name="supplier_rank">1</field>
+    </record>
+    <record id="res_partner_12" model="res.partner" >
+        <field name="name">Belighted</field>
+        <field name="display_name">Belighted</field>
+        <field name="phone">001010101010</field>
+        <field name="email">i@i.am</field>
+        <field name="street">Place de l'Universit&#233; 25</field>
+        <field name="city">Ottignies-Louvain-la-Neuve</field>
+        <field name="zip">1348</field>
+        <field name="country_id" ref="base.be" />
+        <field name="contact_address_complete">Place de l'Universit&#233; 25, 1348 Ottignies-Louvain-la-Neuve, Belgium</field>
+        <field name="category_id" eval="[(6, 0, [])]" />
+        <field name="is_company" eval="True" />
+        <field name="partner_gid">33247</field>
+        <field name="property_stock_customer" ref="stock.stock_location_customers" />
+        <field name="partner_share" eval="True" />
+        <field name="property_stock_supplier" ref="stock.stock_location_suppliers" />
+        <field name="calendar_last_notif_ack">2023-07-29 10:33:12</field>
+        <field name="commercial_company_name">Belighted</field>
+        <field name="message_bounce">1</field>
+        <field name="email_normalized">i@i.am</field>
+        <field name="website">http://belighted.com</field>
+        <field name="ubl_cii_format">ubl_bis3</field>
```

Comment:
check all compute fields most of which are not needed because it's compute base on other value
like `contact_address_complete`
`partner_gid`
`property_stock_customer`
`property_stock_customer`
`calendar_last_notif_ack`